src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -25,6 +25,253 @@
+"""
+
+helps['iot dps access-policy update'] = """
+    type: command
+    short-summary: Update a shared access policy in an Azure IoT Hub device provisioning service.
+    examples:
+        - name: Update access policy 'MyPolicy' in an Azure IoT Hub device provisioning service with EnrollmentWrite right
+          text: >
+            az iot dps access-policy update --dps-name MyDps --resource-group MyResourceGroup --name MyPolicy --rights EnrollmentWrite
+"""
+
+helps['iot dps access-policy list'] = """
+    type: command
+    short-summary: List all shared access policies in an Azure IoT Hub device provisioning service.
+    examples:
+        - name: List all shared access policies in MyDps
+          text: >
+            az iot dps access-policy list --dps-name MyDps --resource-group MyResourceGroup
+"""
+
+helps['iot dps access-policy show'] = """
+    type: command
+    short-summary: Show details of a shared access policies in an Azure IoT Hub device provisioning service.
+    examples:
+        - name: Show details of shared access policy 'MyPolicy' in an Azure IoT Hub device provisioning service
+          text: >
+            az iot dps access-policy show --dps-name MyDps --resource-group MyResourceGroup --name MyPolicy
+"""
+
+helps['iot dps access-policy delete'] = """
+    type: command
+    short-summary: Delete a shared access policies in an Azure IoT Hub device provisioning service.
+    examples:
+        - name: Delete shared access policy 'MyPolicy' in an Azure IoT Hub device provisioning service
+          text: >
+            az iot dps access-policy delete --dps-name MyDps --resource-group MyResourceGroup --name MyPolicy
+"""
+
+helps['iot dps linked-hub'] = """
+    type: group
+    short-summary: Manage Azure IoT Hub Device Provisioning Service linked IoT hubs.
+"""
+
+helps['iot dps linked-hub create'] = """
+    type: command
+    short-summary: Create a linked IoT hub in an Azure IoT Hub device provisioning service.
+    examples:
+        - name: Create a linked IoT hub in an Azure IoT Hub device provisioning service
+          text: >
+            az iot dps linked-hub create --dps-name MyDps --resource-group MyResourceGroup --connection-string
+            HostName=test.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=XNBhoasdfhqRlgGnasdfhivtshcwh4bJwe7c0RIGuWsirW0=
+            --location westus
+        - name: Create a linked IoT hub in an Azure IoT Hub device provisioning service which applies allocation weight and weight being 10
+          text: >
+            az iot dps linked-hub create --dps-name MyDps --resource-group MyResourceGroup --connection-string
+            HostName=test.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=XNBhoasdfhqRlgGnasdfhivtshcwh4bJwe7c0RIGuWsirW0=
+            --location westus --allocation-weight 10 --apply-allocation-policy True
```

Comment:
Secrets in examples should be masked. They triggered a security incident today, 6 years after this PR.